net/ncdc/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                ncdc
-version             1.23.1
+version             1.24.1
 categories          net chat
 platforms           darwin
```

Comment:
this is the default for quite some  time now and thus the "platforms" line should be removed. "port lint" warns about that so you should have seen that when you ran the lint check locally.